Pod/Classes/Device.swift

```diff
@@ -56,4 +58,32 @@ public final class Device: Model,ResponseObject,ResponseCollection {
     public func getExperience() -> Experience?{
         return self.experience
     }
+    
+    /**
+     Get Current Device
+     @return Promise<Device>.
+     */
+    public static func getCurrentDevice() -> Promise<Device>{
+        var prom:Promise<Device>?
+        if let identity:[String:AnyObject] = auth?.get("identity") as! [String:AnyObject] {
+            if let uuididentity = identity["uuid"]{
+             prom = Promise { fulfill, reject in
+                Alamofire.request( Router.getDevice(uuididentity as! String) )
+                    .responseObject { (response: Response<Device, NSError>) in
+                        switch response.result{
+                        case .Success(let data):
+                            fulfill(data)
+                        case .Failure(let error):
+                            return reject(error)
+                        }
+                }
+            }
+            }else{
+                prom = Promise{fulfill,reject in}
```

Comment:
this promise never returns right? we should reject or throw an error if there is an error getting the identity uuid